CHANGES.md

```diff
@@ -1,5 +1,7 @@
 # git master
 
+* [486](https://github.com/Eyescale/CMake/pull/486):
+  Fix coverage report generation for non-subprojects
```

Comment:
We have to agree on the nomenclature, because what you call the subproject case (viz/stable) for me it's the superproject case.